addons/account_peppol/models/res_company.py

```diff
@@ -71,6 +71,7 @@ class ResCompany(models.Model):
         ],
         string='PEPPOL status', required=True, default='not_registered',
     )
+    is_active_on_peppol_proxy = fields.Boolean(compute='_compute_is_active_on_peppol_proxy')
```

Comment:
Do you need to implement search for this field? (seems needed by `peppol_get_message_status` in `account.journal`)